server/odc-core/src/main/resources/i18n/ErrorMessages.properties

```diff
@@ -4,174 +4,174 @@
+com.oceanbase.odc.ErrorCodes.FileCreateUnauthorized=Failed to create the file. Please check disk space and directory permissions.
+com.oceanbase.odc.ErrorCodes.FilePathTraversalDetected=File path traversal detected, please review the request parameter and try again.
+com.oceanbase.odc.ErrorCodes.FileUploading=Uploading the file...
+com.oceanbase.odc.ErrorCodes.FileSuffixNotAllowed=Invalid file extension. File name: {0}.
+com.oceanbase.odc.ErrorCodes.ResourceModifying=The resource is being modified, and concurrent modification is not supported. Please try again later.
+com.oceanbase.odc.ErrorCodes.ResourceCreating=The resource is being created, and concurrent creation is not supported. Please try again later.
+com.oceanbase.odc.ErrorCodes.ResourceSynchronizing={0} is being synchronized. Please try again later.
+com.oceanbase.odc.ErrorCodes.RunningTaskNotTerminable=The current ticket is in execution and cannot be terminated.
+com.oceanbase.odc.ErrorCodes.FinishedTaskNotTerminable=The current ticket is completed and cannot be terminated.
+com.oceanbase.odc.ErrorCodes.TaskNotReadyForDownload=The ticket is not yet completed or has failed. Ticket ID: {0}. Current status: {1}.
+com.oceanbase.odc.ErrorCodes.TaskSqlExecuteFailed=The execution failed on line {0}.
+com.oceanbase.odc.ErrorCodes.TaskLogNotFound=The ticket log with the {0} as {1} does not exist. It might be due to the file expiring or the ticket not being executed.
+com.oceanbase.odc.ErrorCodes.DebugDBMSNotSupported=The DBMS_DEBUG package does not support debugging.
+com.oceanbase.odc.ErrorCodes.DebugDBMSCallFailed=Failed to execute the DBMS_DEBUG package. Details: {0}.
+com.oceanbase.odc.ErrorCodes.DebugPackageCreateFailed=The PL debug package does not exist.
+com.oceanbase.odc.ErrorCodes.DebugTypeUnsupported=Unsupported object type for PL debugging.
+com.oceanbase.odc.ErrorCodes.DebugObjectUnsupported=OBServer does not support debugging the current PL object.
+com.oceanbase.odc.ErrorCodes.DebugSessionNotAlive=The debugging session has expired. Details: {0}.
+com.oceanbase.odc.ErrorCodes.DebugStartFailed=Failed to start debugging. Cause: {0}.
+com.oceanbase.odc.ErrorCodes.DebugInfoParseFailed=Failed to parse the debugging information. Please check if the database version is the latest.
+com.oceanbase.odc.ErrorCodes.DebugTimeout=Debugging timed out. Please close the window and try again.
+com.oceanbase.odc.ErrorCodes.PLObjectFetchFailed=Fail to fetch the DDL of the PL object. Details: {0}.
+com.oceanbase.odc.ErrorCodes.MockError=An error occurred for the mock data ticket. Details: {0}.
+com.oceanbase.odc.ErrorCodes.ImportInvalidZip=Invalid ZIP file. Details: The ZIP file is not exported by ODC or OBDUMPER.
+com.oceanbase.odc.ErrorCodes.ImportInvalidFileType=Invalid file type {0}. Details: Import is not supported for this file type.
+com.oceanbase.odc.ErrorCodes.FlowTaskInstanceExpired=The execution of the workflow timed out.
```

Comment:
user may confused for 3 differenct concept: workflow, ticket and task.
we may unify usage, such as
- while describe as an approve workflow, use 'ticket'
- whiel descrive the job execution, use 'task'